core/sail/shacl/src/test/resources/logback-test.xml

```diff
@@ -6,15 +6,12 @@
 			<pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %msg%n</pattern>
 		</encoder>
 	</appender>
-	
+
+	<logger name="org.eclipse.rdf4j.sail.shacl" level="info"/>
+
 	<root>
-		<level value="info" />
+		<level value="warn" />
```

Comment:
I prefer to keep the shacl logging fairly verbose. I've done a big cleanup for some of the logging now to reduce the output. I've also got a lot of tests set up to only log output if they fail, this greatly reduces the output.

Reply:
Note that logging for anything in package `org.eclipse.rdf4j.sail.shacl` is set to INFO, so you should still get fairly detailed logging for anything shacl-specific.

Reply:
I'd also like to keep the default less verbose. Of course, when a test is giving us problems (intermittent or otherwise), we can always up the verbosity (either locally, or pushed to CI) temporarily to diagnose the issue.

Reply:
I designed the main shacl tests to only output info if they fail. I want to check if your changes affect this or not before your merge the branch. Because of Christmas is might take a few days or a week before I have time to test it. Is that ok @jeenbroekstra ?

Reply:
Sure, no rush. I'm pretty certain it won't be affected as I didn't change anything in the report output writing - I just adapted the actual logging. But doesn't hurt to double-check.

Fwiw I would prefer to have the error reporting on failure also use the logger, instead of printing directly to stdout. But that's a separate issue, and one we can discuss later.

Reply:
I had a look now and your changes look good.